tests/python/pants_test/backend/jvm/tasks/test_jvm_run.py

```diff
@@ -53,6 +53,13 @@ def test_opt_main(self):
     with self.setup_cmdline_run(main=main_entry) as cmdline:
       self.assertTrue(self._match_cmdline_regex(cmdline, main_entry))
 
+  def test_extra_jvm_option(self):
+    options = ['-Dexample.property1=1', '-Dexample.property2=1']
+    with self.setup_cmdline_run(extra_jvm_options=options) as cmdline:
+      for option in options:
+        self.assertTrue(option in cmdline)
+
```

Comment:
This will likely fail to lint due to the extra line here.

Consider installing the commit hooks as described here: https://www.pantsbuild.org/howto_contribute.html#getting-pants-source-code (after installing them, you can skip them whenever you like by running `git commit ... -n`). If you don't install, consider running `build-support/bin/pre-commit.sh` before pushing PRs.

Reply:
Thanks! I somehow didn't think to ask myself where the hooks were, but I'm glad we have them, thanks for the reminder.